$NetBSD: patch-CMakeLists.txt,v 1.1 2013/05/09 12:33:52 fhajny Exp $

sys/wait.h is needed on SunOS and missed by cmake.
--- CMakeLists.txt.orig	2012-08-06 11:42:12.000000000 +0000
+++ CMakeLists.txt
@@ -171,6 +171,7 @@ CHECK_INCLUDE_FILES(sys/time.h   HAVE_SY
 CHECK_INCLUDE_FILES(sys/types.h  HAVE_SYS_TYPES_H)
 CHECK_INCLUDE_FILES(sys/uio.h    HAVE_SYS_UIO_H)
 CHECK_INCLUDE_FILES(sys/un.h     HAVE_SYS_UN_H)
+CHECK_INCLUDE_FILES(sys/wait.h   HAVE_SYS_WAIT_H)
 CHECK_INCLUDE_FILES(time.h       HAVE_TIME_H)
 CHECK_INCLUDE_FILES(unistd.h     HAVE_UNISTD_H)
 CHECK_INCLUDE_FILES(mysql.h      HAVE_MYSQL_H)
